google/cloud/forseti/services/inventory/service.py

```diff
@@ -86,21 +86,20 @@ def Create(self, request, _):
                 last_error=repr(progress.last_error))
 
     @autoclose_stream
-    def List(self, request, _):
+    def list(self, request, _):
         """Lists existing inventory.
 
         Args:
-            request (object): gRPC request object.
             _ (object): Unused.
```

Comment:
Should we add the line "request (object): gRPC request object." back under Args?